client/src/templates/Challenges/components/scene/scene.css

```diff
@@ -37,7 +38,7 @@
 }
 
 .scene-play-btn img {
-  max-width: 75%;
+  transform: translate(5px, 10px);
```

Comment:
the image of the play button isn't fully centered. 

using the transform makes the outline look like (below):

![button_outline](https://github.com/freeCodeCamp/freeCodeCamp/assets/28301511/f45eaa74-e92f-4e26-82db-663750d1d443)